Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -6255,7 +6319,10 @@ - (void)_updateTextInputTraits:(id<UITextInputTraits>)traits
     if ([privateTraits respondsToSelector:@selector(setShortcutConversionType:)])
         privateTraits.shortcutConversionType = _focusedElementInformation.elementType == WebKit::InputType::Password ? UITextShortcutConversionTypeNo : UITextShortcutConversionTypeDefault;
 
-    [self _updateAdditionalTextInputTraits:privateTraits];
+#if HAVE(INLINE_PREDICTIONS)
+    if ([privateTraits respondsToSelector:@selector(setInlineCompletionType:)])
```

Comment:
I don't think we need `respondsToSelector` anymore?

Also, if `inlineCompletionType` exists as API, can we just use `traits` rather than `privateTraits`? If not, I think we need something in `UIKitSPI.h`.